pom.xml

```diff
@@ -70,5 +70,23 @@
             <artifactId>workflow-step-api</artifactId>
             <version>1.15</version>
         </dependency>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>workflow-job</artifactId>
+            <version>1.15</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>workflow-cps</artifactId>
+            <version>2.2</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>workflow-basic-steps</artifactId>
+            <version>1.15</version>
```

Comment:
@oleg-nenashev It did not require a bump -- if we auto-bump to 2.x, it makes sense to do that separately.